### ISSUE TITLE
[VoiceOver] Gutenberg Audit

### ISSUE THREAD

**Issue by shiki** [wordpress-mobile/WordPress-iOS#13225]
WIP.

**Comment by etoledom** [comment]
Great job @shiki !
Can't wait to start working on all these improvements 🤩

**Comment by shiki** [comment]
Thank you, @etoledom! I'm glad I could help. I'm looking forward to the improvements. 🙂

**Comment by rezzap** [comment]
There was some feedback left in an app review regarding unlabeled buttons. They didn't specify which button they are, but I wanted to add it here anyway:
Translation:

> The app basically works very well with the VoiceOver screen reader for the blind, but unfortunately there are a few unlabeled buttons, the voice output says that it is a button, but not which action the button activates. A blind person only hears the word “button”, but in the end does not know what, what command this button executes after tapping it. 

Original: 

> Die App funktioniert grundsätzlich ganz gut mit dem Screenreader VoiceOver für Blinde, aber es gibt leider ein paar unbeschriftete Buttons, die Sprachausgabe sagt zwar an, dass es eine Taste ist, aber nicht, welche Aktion die Taste, der Button aktiviert. Ein Blinder hört also nur das Wort „Taste“, weiß aber im Endeffekt nicht, was, welchen Befehl diese Taste nach antippen ausführt. 

Reference iOS review by hebejacq – Edited Nov 26, 2020
Version 16.1.1

**Comment by zdenys** [comment]
Reported in 5219873-zd-woothemes

### Image caption editing does not work with Apple voice control

>1. Apple voice control features for editing text (navigation, selection, editing, deletion) work consistently with paragraph blocks but do not work for editing captions for images
>2. Voice control commands are visible from the Settings/Accessibility/Voice Control as shown below

![image](https://user-images.githubusercontent.com/39308239/168778156-ebc464a1-a3cf-405d-80fc-ed4413b26392.png)

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by jkmassel** [comment]
We're replacing Gutenberg Mobile with GutenbergKit, so we won't end up addressing this.
